src/Microsoft.Azure.WebJobs.Logging/LogFactory.cs

```diff
@@ -15,23 +13,35 @@ public static class LogFactory
         /// <summary>
         /// Get a reader that reads from the given table. 
         /// </summary>
-        /// <param name="table"></param>
+        /// <param name="tableLookup"></param>
         /// <returns></returns>
-        public static ILogReader NewReader(CloudTable table)
+        public static ILogReader NewReader(ILogTableProvider tableLookup)
         {
-            return new LogReader(table);
+            return new LogReader(tableLookup);
         }
 
         /// <summary>
         /// Create a new writer for the given compute container name that writes to the given table.
         /// Multiple compute instances can write to the same table simultaneously without interference. 
         /// </summary>
         /// <param name="computerContainerName">name of the compute container. Likley %COMPUTERNAME%. </param>
-        /// <param name="table">underlying azure storage table to write to.</param>
+        /// <param name="tableLookup">underlying azure storage table to write to. 
+        /// Passed a string suffix (which will consist of valid Azure table characters). 
```

Comment:
I don't understand what "passed a string suffix" refers to. Please clean up the xml comments for these public types